user-interface/src/main/java/life/qbic/datamanager/views/register/UserRegistrationOrcIdComponent.java

```diff
@@ -0,0 +1,213 @@
+package life.qbic.datamanager.views.register;
+
+import com.vaadin.flow.component.ComponentEvent;
+import com.vaadin.flow.component.ComponentEventListener;
+import com.vaadin.flow.component.Key;
+import com.vaadin.flow.component.button.Button;
+import com.vaadin.flow.component.button.ButtonVariant;
+import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.component.html.H2;
+import com.vaadin.flow.component.textfield.TextField;
+import com.vaadin.flow.data.binder.Binder;
+import com.vaadin.flow.data.binder.ValidationResult;
+import com.vaadin.flow.data.binder.Validator;
+import com.vaadin.flow.data.validator.EmailValidator;
+import com.vaadin.flow.data.validator.RegexpValidator;
+import com.vaadin.flow.spring.annotation.SpringComponent;
+import com.vaadin.flow.spring.annotation.UIScope;
+import jakarta.validation.constraints.NotEmpty;
+import java.io.Serial;
+import java.io.Serializable;
+import java.util.Objects;
+import life.qbic.datamanager.views.notifications.ErrorMessage;
+import life.qbic.identity.api.UserInformationService;
+
+/**
+ * User Registration OrcId Component
+ * <p>
+ * Card Stylized component similar to {@link UserRegistrationComponent} component, Providing the
+ * input fields with validation during user registration if done via OrcId. This skips the password
+ * validation since the user id validated via her OrcId
+ */
+@SpringComponent
+@UIScope
+public class UserRegistrationOrcIdComponent extends Div {
+
+  @Serial
+  private static final long serialVersionUID = 822067456152829562L;
+
+  /*Validation via Binder does not show error message if an email field is used, see
+   * https://github.com/vaadin/flow-components/issues/4618 for details */
+  private final TextField email = new TextField("Email");
+
+  private final TextField fullName = new TextField("Full Name");
+
+  private final TextField username = new TextField("Username");
+
+  private final Button registerButton = new Button("Register");
+
+  private final Div notificationLayout = new Div();
+
+  private final H2 titleSpan = new H2("Add your details");
+
+  private final Div description = new Div();
+
+  private final Binder<UserRegistrationOrcIdInformation> registrationInformationBinder = new Binder<>(
+      UserRegistrationOrcIdInformation.class);
+
+  private final transient UserInformationService userInformationService;
+
+  public UserRegistrationOrcIdComponent(UserInformationService userInformationService) {
+    this.userInformationService = Objects.requireNonNull(userInformationService);
+    addClassName("user-registration-component");
+    registerButton.addThemeVariants(ButtonVariant.LUMO_PRIMARY);
+    username.setHelperText("Your unique user name, visible to other users");
+    description.add(
+        "Since your full name and email are not shared by ORCiD, we need some additional information from you");
```

Comment:
```suggestion
        "Please complete missing information to create an account with us.");
```
```java
"Since your full name and email are not shared by ORCiD, we need some additional information from you"
```
This is not true. The name is shared if this is enabled by the user in her/his orcid settings.